services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -278,6 +278,16 @@ export const ReleaseDialog: React.FC<ReleaseDialogProps> = (props) => {
                                 <span className={classNames('release-dialog-commitMessage', className)}>
                                     {release?.sourceMessage}
                                 </span>
+                                &nbsp;
+                                <span className={classNames('release-dialog-display-version', className)}>
```

Comment:
this is not the right spot to display it. The commit Id is already displayed in line 308 (below). Please add your change there.

And because it will then get rather complicated, please extract that into a separate component (to hide this "if-then-else" logic). This component should also take care of the "title" attribute, as that one will be more complex as well.
(I'm referring to this sentence in the story "If we display the display version, a tooltip should show the commit Id.")